### PR TITLE
JP-374: ZEROFRAME processing to STCAL changed function signatures.

### PR DESCRIPTION
**Pull request by kmacdonald-stsci**
…n STCAL.

GitHub issue, Closes #

Resolves [JP-374](https://jira.stsci.edu/browse/JP-374)

**Description**

The ticket JP-374 got merged into STCAL, but changed a saturation function signature causing failures in romancal.  The function signature in romancal has been updated and the saturation unit tests pass.

Checklist
- [ ] Tests

- [ ] Documentation

- [ ] Change log

- [ ] Milestone

- [ ] Label(s)

### CODE REVIEW

**Review by ddavis-stsci — APPROVED** [review]
LGTM

**Review by cshanahan1 — APPROVED** [review]
Looks fine here, just add the 'zframe' to the returns section in the docstring on stcal

File: romancal/saturation/saturation.py [hunk 1]
```diff
@@ -46,9 +46,9 @@ def flag_saturation(input_model, ref_model):
     sat_dq = ref_model.dq.copy()
 
     # Obtain dq arrays updated for saturation
-    gdq_new, pdq_new = flag_saturated_pixels(data, gdq, pdq, sat_thresh,
-                                             sat_dq, ATOD_LIMIT, dqflags.pixel,
-                                             n_pix_grow_sat=0)
```
Comment (nden) [inline romancal/saturation/saturation.py:51]: Please add a comment here explaining why this is necessary, e.g. something like `the third returned value is the zero frame which is not used by romancal and is always None`.

**Comment by nden** [comment]
Needs a change log entry.
And set the version in setup.cfg `stcal>=0.7`

File: romancal/saturation/saturation.py (hunk 1 above)
Comment (kmacdonald-stsci) [reply romancal/saturation/saturation.py:51]: I added this comment.

**Comment by nden** [comment]
@kmacdonald-stsci There's a conflict now that needs to be resolved and also add a change log

**Review by nden — APPROVED** [review]

**Pull request merged**